### DISCUSSION TITLE
How to make edex-ui automaticly at start

### DISCUSSION THREAD

**Discussion by S1700** [GitSquared/edex-ui#1099] [Questions]
How do i make edex-ui run automatically when i login if its even possible 

Thats it rly

Thanks

**Comment by S1700** [comment]
nvm i got it
if any one wants to know how i did it just ask :)

**Reply by h-bag** [reply]
or you could just... post it here? in case someone years down the line wants to know

**Reply by S1700** [reply]
make a bash file with this in:
```
cd [edex-ui dir here]
npm run start
```
save it as a .sh file
then move that file into /etc/profile.d/
done!
